submissions/adeyemiJuwon/todo/src/components/Form.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { IoIosAdd } from 'react-icons/io';
```

Comment:
There is no need to install a package for a plus icon. You can add the `+` in the content just like you did for the check.